.github/workflows/docs.yml

```
@@ -0,0 +1,49 @@
+        run: npm run build:ts
+
+      - name: Generate documentation
+        run: npm run docs
+
+      - name: Copy artifacts
+        run: |
+          mkdir -p site/dist
+          cp dist/* site/dist/
+
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)